src/Documents/Import/PageImporter.php

```diff
@@ -20,20 +21,37 @@ public function __construct(
     ) {
     }
 
-    public function parseYaml(string $yamlInput, bool $forcedSave = true): mixed
+    /**
+     * @return array<Page>
+     *
+     * @throws ConverterException
+     * @throws DuplicateFullPathException
+     */
+    public function parseYaml(string $yamlInput, bool $forcedSave = true): array
```

Comment:
`importYaml` wäre einheitlicher zum Exporter. Oder `fromYaml`. Hauptsache einheitlich.

Reply:
Same comment: Tried to avoid breaking changes.